Kevin Williams PA1

```diff
@@ -0,0 +1,365 @@
+# COMP3410 Program Template
+# Author: Kevin Williams
+# Assignment: PA[X]
+# Date: Date of submission
+
+# Turn in one .asm file per assignment component
+# Remember to submit it as a pull request to the GitHub repo for the assignment
+
+# fibonacci.asm
+# Input: None
+# Output: Print to terminal N Fibonacci numbers
+
+##########################################################
+# .data segment. Constant and variable definitions go here.
+##########################################################
+
+	.data
+	
+
+fibs:	.word   0 : 9         # create an array variable named "fibs" of 9 word-length elements (4 bytes each)
+size:	.word  9              # create a single integer variable named "size" that indicates the length of the array
+text:   .asciiz " Enter any number thats between 1-9."
+###############################################
+# .text segment. Assembly instructions go here.
+###############################################
+	.text
+	      la   $s0, fibs        # load address of array into $s0
+	      #la $s5, size         # load address of size variable into $s5
+	      #lw $s5, 0($s5)       # load array size from its address in the regisgter
+	      
+	      li $v0, 4             # 4 =print string
+	      la $a0, text          # load the address of text
+	      syscall               # do the printing
+	      
+	      li $v0, 5             # 5=read int
+	      syscall               # read the int, result stored in $v0
+	      move $s5, $v0
```

Comment:
-5 Accepts input greater than 9.  This allows storing numbers beyond the bounds of the array.